src/components/content/systemStatus/useSystemStatusQuery.ts

```diff
@@ -0,0 +1,16 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ * SPDX-FileCopyrightText: Huawei Inc.
+ */
+
+import { useQuery } from '@tanstack/react-query';
+import { AdminService } from '../../../xpanse-api/generated';
+import { systemStatusQueryKey } from './queryKey';
+
+export function useSystemStatusQuery() {
+    return useQuery({
+        queryKey: [systemStatusQueryKey],
```

Comment:
systemStatusQueryKey -> set it to any unique string. `healthcheck` is a good value.